src/lib/identity/api.ts

```diff
@@ -74,41 +91,126 @@ const getUserFromCookie = (): IdentityUserFromCache => {
 	return userFromCookieCache;
 };
 
-const isUserLoggedIn = (): boolean => getUserFromCookie() !== null;
-
-const fetchUserIdentifiers = () => {
-	const url = `${idApiRoot}/user/me/identifiers`;
-	return fetch(url, {
+/**
+ * Fetch the logged in user's Google Tga ID from IDAPI
+ * @returns one of:
+ * - string - the user's Google Tag ID
+ * - null - if the request failed
+ */
+const fetchGoogleTagIdFromApi = (): Promise<string | null> =>
+	fetch(`${idApiRoot}/user/me/identifiers`, {
 		mode: 'cors',
 		credentials: 'include',
 	})
 		.then((resp) => {
 			if (resp.status === 200) {
-				return resp.json();
+				return resp.json() as Promise<{ googleTagId: string }>;
 			} else {
-				console.log(
-					'failed to get Identity user identifiers',
-					resp.status,
-				);
-				return null;
+				throw resp.status;
 			}
 		})
+		.then((json) => json.googleTagId)
 		.catch((e) => {
 			console.log('failed to get Identity user identifiers', e);
 			return null;
 		});
-};
 
-const getUserIdentifiersFromApi = mergeCalls(
-	(mergingCallback: (u: IdentityUserIdentifiers | null) => void) => {
+const isUserLoggedIn = (): boolean => getUserFromCookie() !== null;
+
+const getAuthStatus = async (): Promise<AuthStatus> => {
+	if (isInOktaExperiment) {
+		const { isSignedInWithOktaAuthState } = await import('./okta');
+		const authState = await isSignedInWithOktaAuthState();
+		if (authState.isAuthenticated) {
+			return {
+				kind: 'SignedInWithOkta',
+				accessToken: authState.accessToken,
+				idToken: authState.idToken,
+			};
+		} else {
+			return {
+				kind: 'SignedOutWithOkta',
+			};
+		}
+	} else {
 		if (isUserLoggedIn()) {
-			void fetchUserIdentifiers().then((result) =>
-				mergingCallback(result),
-			);
+			return {
+				kind: 'SignedInWithCookies',
+			};
 		} else {
-			mergingCallback(null);
+			return {
+				kind: 'SignedOutWithCookies',
+			};
 		}
-	},
-);
+	}
+};
+
+const isUserLoggedInOktaRefactor = (): Promise<boolean> =>
```

Comment:
We've renamed this to `isUserLoggedIn` in Frontend - we only had a seperate method whilst we were still migrating so we could run okta and non okta in parallel, but if everything already uses the Okta method you mayaswell remove the non-okta one and rename the Okta one.